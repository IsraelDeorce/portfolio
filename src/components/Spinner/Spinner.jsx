function Spinner() {
  return (
    <div>LOADING...</div>
  )
}

export default Spinner