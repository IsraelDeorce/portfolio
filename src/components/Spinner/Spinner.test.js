import Spinner from './Spinner'

describe('<Spinner/>', () => {
  describe('Snapshot', () => {
    it('renders without crashing using shallow', () => {
      expect(shallow(<Spinner />)).toMatchSnapshot()
    })
  })
})
