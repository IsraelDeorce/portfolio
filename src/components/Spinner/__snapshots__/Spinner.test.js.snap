// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> Snapshot renders without crashing using shallow 1`] = `
<div>
  Loading...
</div>
`;
